### PR TITLE
Get possible responses from i18n file (for question 1)

### PR DESCRIPTION
**Pull request by brucebolt**
Removing the hardcoding of specific response options and simply loading all the options included in the i18n YAML file for this question will make it simpler to change the questions/answers.

### CODE REVIEW

File: app/views/coronavirus_form/medical_equipment.html.erb [hunk 1]
```diff
@@ -17,18 +17,13 @@
...
-    },
-    {
-      value: t('coronavirus_form.medical_equipment.options.option_no.label'),
-      text: t('coronavirus_form.medical_equipment.options.option_no.label'),
-      checked: session[:medical_equipment] == t('coronavirus_form.medical_equipment.options.option_no.label'),
-    },
-  ]
+      value: item[:label],
+      text: item[:label],
+      checked: session[:hotel_rooms] == item[:label],
```
Comment (issyl0) [inline app/views/coronavirus_form/medical_equipment.html.erb]:
```suggestion
      checked: session[:medical_equipment] == item[:label],
```

:question:

Comment (brucebolt) [reply app/views/coronavirus_form/medical_equipment.html.erb]: Oops!  Copy and pasted this from another view.

Comment (issyl0) [reply app/views/coronavirus_form/medical_equipment.html.erb]: This is why we have code review especially at this time of day. :laughing:

**Review by issyl0 — APPROVED** [review]

File: app/views/coronavirus_form/medical_equipment.html.erb (hunk 1 above)
Comment (brucebolt) [reply app/views/coronavirus_form/medical_equipment.html.erb]: Couldn't agree more!

**Pull request merged**